examples/chat/tui/commands.nim

```diff
@@ -27,6 +27,62 @@ const hashCharSet = {'#'}
 # `command` proc should implement the appropriate logic to deal with those
 # values
 
+# AddCustomToken ----------------------------------------------------------------
+
+proc help*(T: type AddCustomToken): HelpText =
+  let command = "addcustomtoken"
+  HelpText(command: command, aliases: aliased[command], parameters: @[
+    CommandParameter(name: "address", description: "Address of the " &
+      "custom token."),
+    CommandParameter(name: "name", description: "Name of " &
+      "the custom token."),
+    CommandParameter(name: "symbol", description: "Symbol of the " &
+      "custom token."),
+    CommandParameter(name: "color", description: "(Optional) Color (in hex) for " &
+      "custom token."),
+    CommandParameter(name: "decimals", description: "(Optional) Number of decimals to use for " &
+      "the custom token.")
+    ], description: "Creates a new custom token ")
+
+proc new*(T: type AddCustomToken, args: varargs[string]): T =
+  var
+    address = ""
+    name = ""
+    symbol = ""
+    color = ""
+    decimals = ""
+  if args.len > 2:
+    address = args[0]
+    name = args[1]
+    symbol = args[2]
+  if args.len > 4:
+    color = args[3]
+    decimals = args[4]
+
+  T(address: address, name: name, symbol: symbol, color: color, decimals: decimals)
+
+proc split*(T: type AddCustomToken, argsRaw: string): seq[string] =
+  argsRaw.split(" ")
+
+proc command*(self: ChatTUI, command: AddCustomToken) {.async, gcsafe,
+  nimcall.} =
+
+  try:
+    if command.address == "":
+      self.wprintFormatError(epochTime().int64,
```

Comment:
We believe we should try to move away from using `epochTime().int64` and instead use `getTime().toUnix` from the `std/times` lib.

Reply:
Fixed.